openwebstart/src/main/java/com/openwebstart/launcher/OwsJvmLauncher.java

```diff
@@ -156,6 +157,13 @@ private void launchExternal(
 
     }
 
+    private void checkForJava9Arg(final List<String> javawsArgs) {
+        final String result = javawsArgs.stream().filter(arg -> JvmUtils.isValidStartingJavaModulesArgument(arg)).collect(Collectors.joining(","));
```

Comment:
Stream offers a method `boolean anyMatch(Predicate)`
If you use this instead of `filter` and `collect` the code becomes much shorter

Reply:
yes but i wanted to show all the problematic arguments.